src/sestest/ServerOptions.cs

```diff
@@ -126,13 +146,15 @@ public ServerOptions WithAudience(string audience)
         /// <param name="connectionCertificate">Certificate to use for our HTTPS connection (optional).</param>
         /// <param name="serverUrl">Server host URL (optional).</param>
         /// <param name="audience">Audience expected of tokens.</param>
+        /// <param name="issuer">Issuer expected of tokens.</param>
```

Comment:
"Issuer expected of tokens." reads oddly to me.  "Expected issuer of tokens" perhaps?